### DISCUSSION TITLE
How to export output in command line

### DISCUSSION THREAD

**Discussion by wxystudio** [APSIMInitiative/ApsimX#8793] [General]
Hello, now I can successfully run the apsim model in command line following this script: https://apsimnextgeneration.netlify.app/usage/commandlineusage/
And when I use the "--csv", I can export a .csv file to my /publish/ folder, like this:
![image](https://github.com/APSIMInitiative/ApsimX/assets/17384283/14ad7a29-470d-4846-818d-101b73b62b70)
![image](https://github.com/APSIMInitiative/ApsimX/assets/17384283/d9f92508-82a3-4068-ad15-215e511a222d)

But how to specify the output file name and directory? 
In my opinion, I want to run commands and output a .csv file in a specific folder, like "--csv /myfolder/result.csv". could you please tell me how to do it?

**Comment by hol353** [comment]
It is not currently possible. 

> In my opinion, I want to run commands and output a .csv file in a specific folder, like "--csv /myfolder/result.csv"

You are assuming that you only have one report in your simulation which you want outputted to one .csv file. In my simulations I usually have many reports, resulting in many .csv files.

We could potentially change the --csv switch to optionally let you specify an output directory like  ```--csv /myfolder``` and the file names are based on the names of the report files. e.g. if you have a report called 'report' then you get ```/myfolder/result.csv```

**Reply by wxystudio** [reply]
thank you!
